pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -356,32 +356,34 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	var command []string
 	var args []string
 	// If the OSD was prepared with ceph-volume and running on PVC and using the LVM mode
-	if osdProps.onPVC() && osd.CVMode == "lvm" {
-		// if the osd was provisioned by ceph-volume, we need to launch it with rook as the parent process
-		command = []string{path.Join(rookBinariesMountPath, "rook")}
-		args = []string{
-			"ceph", "osd", "start",
-			"--",
-			"--foreground",
-			"--id", osdID,
-			"--fsid", c.clusterInfo.FSID,
-			"--cluster", "ceph",
-			"--setuser", "ceph",
-			"--setgroup", "ceph",
-			fmt.Sprintf("--crush-location=%s", osd.Location),
-		}
-		osd.LVBackedPV = true
-	} else if osdProps.onPVC() && osd.CVMode == "raw" {
-		doBinaryCopyInit = false
-		doConfigInit = false
-		command = []string{"ceph-osd"}
-		args = []string{
-			"--foreground",
-			"--id", osdID,
-			"--fsid", c.clusterInfo.FSID,
-			"--setuser", "ceph",
-			"--setgroup", "ceph",
-			fmt.Sprintf("--crush-location=%s", osd.Location),
+	if osdProps.onPVC() {
+		if osd.CVMode == "lvm" {
+			// if the osd was provisioned by ceph-volume, we need to launch it with rook as the parent process
+			command = []string{path.Join(rookBinariesMountPath, "rook")}
+			args = []string{
+				"ceph", "osd", "start",
+				"--",
+				"--foreground",
+				"--id", osdID,
+				"--fsid", c.clusterInfo.FSID,
+				"--cluster", "ceph",
+				"--setuser", "ceph",
+				"--setgroup", "ceph",
+				fmt.Sprintf("--crush-location=%s", osd.Location),
+			}
+			osd.LVBackedPV = true
+		} else {
+			doBinaryCopyInit = false
```

Comment:
How about adding a comment that this is for "raw"?

Reply:
Indeed,added.